### DISCUSSION TITLE
How to supplement documentation?

### DISCUSSION THREAD

**Discussion by WalkWeb** [Codeception/Codeception#6658] [General]
In official [documentation](https://codeception.com/docs/reference/Commands) command not mentioned:

`codecept run Unit --filter=NameMethodTest`

it is much more convenient than:

`codecept run Unit Namespace/ClassName:NameMethodTest`

Because you need to specify full path to class.

How can I supplement documentation?

**Comment by Naktibalda** [comment]
The shortest version is `codecept run Unit :NameMethodTest`.

If I remember correctly, test filters are documented in two places.
1. https://codeception.com/docs/GettingStarted#Running-Tests
2. https://codeception.com/docs/reference/Commands#Run

GettingStarted page can be updated by editing https://github.com/Codeception/codeception.github.com/blob/master/docs/GettingStarted.md
Commands page can be updated by editing docblock at https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Command/Run.php#L44-L127
